tests/test_servo.py

```diff
@@ -66,6 +66,22 @@ def skip_if_monitoring_is_not_available(servo):
         pytest.skip("Monitoring is not available")
 
 
+class SDOReadTimeoutManager:
+    def __init__(self, network, new_value):
+        self.__net = network
+        self.__new_value = new_value
+
+    def __enter__(self):
+        self.__set_sdo_read_timeout(self.__new_value)
+        return self
+
+    def __set_sdo_read_timeout(self, value_s):
+        self.__net._ecat_master.sdo_read_timeout = int(1_000_000 * value_s)
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        self.__set_sdo_read_timeout(self.__net.DEFAULT_ECAT_CONNECTION_TIMEOUT_S)
```

Comment:
It should be set to the previous timeout, not the default

Reply:
The previous timeout is the default timeout. I did it like this because the `sdo_read_timeout` attribute cannot be read.

Reply:
Why couldn't it be read? it is a public python attribute
![image](https://github.com/user-attachments/assets/d2137540-d825-4db8-92cb-26be43c01368)

Reply:
Done